src/app/pages/resources/api/chpl-api.scss

```diff
@@ -4,26 +4,82 @@
         color: #333;
     }
 
-    .endpoint {
+    .btn.authorize{
```

Comment:
We don't want to keep these styles in the scss file. This file is going away. These styles should be either in a specific scss file that's scoped (and named) to the new swagger component, or inside one of the useStyles blocks in the api.jsx file